packages/backend/src/types.ts

```diff
@@ -30,4 +31,5 @@ export type PluginEnvironment = {
   config: Config;
   reader: UrlReader;
   discovery: PluginEndpointDiscovery;
+  tokenManager: TokenManager;
```

Comment:
Is there an expectation that this TokenManager would eventually work with both backend and frontend tokens? If not, I wonder if we should rename the property to `serverTokenManager` (but leave the type as `TokenManager`).